src/pages/dbot/index.tsx

```diff
@@ -14,14 +14,31 @@ import { isBrowser } from 'common/utility'
 import BackgroundPatternDBot from 'images/common/bg_banner_signup.png'
 import DHero from 'components/custom/_dhero.js'
 import DNumber from 'components/custom/_dnumbers.js'
-const DBotVideo = Loadable(() => import('./_dbot-video.js'))
+import { MetaAttributesType } from 'types/page.types'
+import { ROW, UKEU } from 'components/containers/visibility'
```

Comment:
Please import from line 6 and remove EU, NonEU if not in used.

Reply:
@pavel-latyshou-deriv Please also import `ROW` and `UKEU` from containers instead of visibility as mentioned above.